File: src/main/java/um/tesoreria/report/hexagonal/chequeras/domain/model/Facultad.java
package um.tesoreria.report.hexagonal.chequeras.domain.model;

import lombok.*;
import java.math.BigDecimal;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Facultad {
    private Integer facultadId;
    private String nombre;
    private String codigoempresa;
    private String server;
    private String dbadm;
    private String dsn;
    private BigDecimal cuentacontable;
    private String apiserver;
    private Long apiport;
}

File: src/main/java/ar/edu/um/tesoreria/factura/rest/kotlin/model/Facultad.kt
package ar.edu.um.tesoreria.factura.rest.kotlin.model

import ar.edu.um.tesoreria.factura.rest.model.Auditable
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import java.math.BigDecimal

@Entity
@Table
data class Facultad(

    @Id
    @Column(name = "fac_id")
    var facultadId: Int? = null,

    @Column(name = "fac_nombre")
    var nombre: String = "",

    @Column(name = "fac_codigo_empresa")
    var codigoempresa: String = "",

    @Column(name = "fac_server")
    var server: String = "",

    @Column(name = "fac_db_adm")
    var dbadm: String = "",

    @Column(name = "fac_dsn")
    var dsn: String = "",

    @Column(name = "fac_pla_cuenta")
    var cuentacontable: BigDecimal = BigDecimal.ZERO,

    @Column(name = "api_server")
    var apiserver: String = "",

    @Column(name = "api_port")
    var apiport: Long = 0L

) : Auditable()
